konami/kpi.md

```diff
@@ -0,0 +1,190 @@
+# Konami Product Identifier (KPI)
+
+a.k.a. "mcodes"
+
+For identification purpose, Konami labels their products, e.g. full cabinet assembly, hardware +
+software upgrade kit, software update, using a propriatary format. These "product codes" are often
+seen denoting specific versions/iterations of software or hardware.
+
+## Sources
+
+* pigstall
+* [bemaniwiki](http://bemaniwiki.com/)
+* Game/cabinet manuals
+
+## Product identifier format
+
+Format string: `ptggg:d:s:r:e`
+
+* `p`: product type
+  * `G`: game (?)
+* `t`: license type/eamuse participation
```

Comment:
We should probably clarify the uncertainty of the meaning of this.

Reply:
Reached out to some folks who might be able to help us out here.